docs/JSON-RPC-API/trace-module-overview.md

```diff
@@ -0,0 +1,53 @@
+# "trace" Module Overview
+
+The trace module is for getting a deeper insight into transaction processing. It includes two sets of calls; the transaction trace filtering API and the ad-hoc tracing API. You can find the documentation for the supported methods [here](/JSON-RPC-API/modules/trace/).
```

Comment:
Some context here regarding the relationship of `trace_` to `debug_` would, I think, be helpful.